src/layouts/home.njk

```diff
@@ -7,33 +7,43 @@ layout: base
     <div class="intro">
       <img src="images/neume-icon-black.svg"/>{{ content | safe }}
     </div>
-
-    {% if site.socialLinks %}
-      <ul class="hero__social-links">
-        {% for item in site.socialLinks %}
-          {% set externalAttribute = '' %}
-
-          {% if item.external %}
-            {% set externalAttribute = 'target="_blank" rel="noopener noreferrer"' %}
-          {% endif %}
-
-          <li>
-            <a href="{{ item.url | url }}" {{ externalAttribute | safe }}>
-              <img class="icon-white-mask" src="images/{{item.text}}-icon.svg"/>
-            </a>
-          </li>
-        {% endfor %}
-      </ul>
-    {% endif %}
+    {% if site.socialLinks %}{% endif %}
   </div>
-
 </header>
-
-<section class="post-list__wrapper">
-
-  {% set postslist = collections.posts | head(-3) %}
-  {% include "posts-list.njk" %}
-
-</section>
-
-<a class="all-posts-link" href="{{ '/blog/' | url }}">View All Posts</a>
\ No newline at end of file
+<section class="stats-wrapper">
+  <section class="stats">
+    <div class="b">INDEXED TRACKS</div>
+    <div class="inverted b">{{ musicos.trackCount }}</div>
+    <br/>
```

Comment:
not needed (at least when using css grid)

```suggestion
```

Reply:
The CSS grid solution looks good to me, I noticed that the table broke when on mobile also